src/Util/Reporter.php

```diff
@@ -14,40 +14,47 @@ class Reporter
     /**
      * @param array $stats
      */
-    public static function report($stats)
+    public static function report(array $stats)
     {
-        $send = function () use ($stats) {
-            if (extension_loaded('curl')) {
-                $body = json_encode($stats, JSON_PRETTY_PRINT);
-                $ch = curl_init(self::ENDPOINT);
-                curl_setopt($ch, CURLOPT_HTTPHEADER, [
-                    'Content-Type: application/json',
-                    'Content-Length: ' . strlen($body)
-                ]);
-                curl_setopt($ch, CURLOPT_CUSTOMREQUEST, 'GET');
-                curl_setopt($ch, CURLOPT_POSTFIELDS, $body);
-                curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
-                curl_setopt($ch, CURLOPT_FOLLOWLOCATION, true);
-                curl_setopt($ch, CURLOPT_MAXREDIRS, 10);
-                curl_setopt($ch, CURLOPT_CONNECTTIMEOUT, 5);
-                curl_setopt($ch, CURLOPT_TIMEOUT, 5);
-                curl_exec($ch);
-            } else {
-                file_get_contents(self::ENDPOINT . '?' . http_build_query($stats));
-            }
-        };
-
-
+        $pid = null;
         if (extension_loaded('pcntl')) {
             declare(ticks = 1);
             $pid = pcntl_fork();
-            if ($pid === 0) {
-                posix_setsid();
```

Comment:
@elfet 
> posix_setsid — Make the current process a session leader.
So a child process becomes a parent.

My test script:
```
<?php

$pid = null;
if (extension_loaded('pcntl')) {
    declare(ticks = 1);
    $pid = pcntl_fork();
}

if (is_null($pid) || $pid === -1) {
    print_r('Parent');
} elseif ($pid === 0) {
    sleep(10);
    file_put_contents('test.txt', 'Child');
    exit(0);
}

print_r('Final!');
```
What I see is fast printing "Final!" (without "Parent") and finishing script. In 10 seconds `test.txt` file appears with "Child!" content.